### PR TITLE
bat files for Windows

### PR DESCRIPTION
**Pull request by AuraDJ**
Created some bat files for cmd illiterate.

### CODE REVIEW

**Review by ArchLeaders — CHANGES_REQUESTED** [review]
Can you add something to the readme to warn that these scripts will always write v7 LE?

**Comment by AuraDJ** [comment]
I added a warning

File: bat files/to byml warning.md [hunk 1]
```diff
@@ -0,0 +1 @@
+"all yaml to byml.bat" will always create version 7 little endian byml (totk specifications). Open in text editor to change parameters.
```
Comment (ArchLeaders) [inline bat files/to byml warning.md]: This isn't the readme, odds are nobody will look at this file.

```suggestion

```

**Review by ArchLeaders — COMMENTED** [review]
The warning should be added to the readme, and the file names should not have spaces. I can make these changes though.

**Review by ArchLeaders — APPROVED** [review]

**Pull request merged**